src/Catty/DataModel/UserData/UserList.swift

```diff
@@ -110,4 +110,37 @@
         }
         return result
     }
+
+    func stringRepresentation() -> String {
+        var value = ""
+        if !self.isEmpty {
+            var allElemenetsAreSilgleLength = true
+            var elements = [String]()
+
+            for index in 1...self.count {
+                var newValue = ""
+
+                if let listElement = self.element(at: index) {
+                    if let stringElem = listElement as? String {
```

Comment:
To use consistent variable names, please name them stringElement, intElement an doubleElement